test/units/testsuite-50.sh

```diff
@@ -962,6 +988,7 @@ check_usual_suspects_after_unmerge() {
 fake_root=${fake_roots_dir}/simple-read-only-with-read-only-hierarchy
 hierarchy=/usr
 
+move_env_out
```

Comment:
Why are all of these being added? Seems unnecessary? Also the commit message is outdated

Reply:
This is to make sure that the test cases are not affected by environment variables, if such happen to leak here. Also updated the commit message.

Reply:
Why would it leak? The test is started as a unit so the env is fresh. Please drop all of these, as they add a lot of noise and boilerplate.

Reply:
Dropped.